src/components/docs/ThemeToggle/ThemeToggle.tsx

```diff
@@ -1,22 +1,16 @@
-import { useCallback, useState } from "react";
+import { useCallback } from "react";
 import cn from "clsx";
 
+import useDarkTheme from "@/hooks/useDarkTheme";
+
 import { MoonIcon } from "./MoonIcon";
 import { SunIcon } from "./SunIcon";
 
 export const ThemeToggle = () => {
-  const [isDark, setIsDark] = useState(false);
+  const [isDark, setIsDark] = useDarkTheme();
 
   const toggleTheme = useCallback(() => {
```

Comment:
```suggestion
  const toggleTheme = useCallback(() => {
    setIsDark(state => !state)
  }, []]
```

@CaCaBlocker You can do this to remove the deps

Reply:
Yeah, you're right. I just updated it.

Reply:
@EiffelFly I just added deps. Till now, I tried to use npm(install & lint), not pnpm. Certainly, this project is working well with pnpm.